tests/unit/GIRUtilsTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+use Drupal\user\Entity\User;
+use Drupal\taxonomy\Entity\Term;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Handler\MockHandler;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Psr7\Response;
+
+use Drupal\os2forms_egir\GIRUtils;
+
+
+class GIRUtilsTest extends \Codeception\Test\Unit
+{
+  /**
+   * @var \UnitTester
+   */
+  protected $tester;
+  protected $utils;
+    
+    protected function _before()
```

Comment:
Vi kunne overveje, om vi skal mocke HTTP clienten med det samme. Den har som før nævnt ikke nogen indivirkning på vores andre utilities. Måske igen et argument for at splitte det op.

Reply:
God pointe, det vil jeg lige overveje.

Reply:
Sådan som GuzzleHttp fungerer med MockHandler er man faktisk nødt til at have handleren, før man initialiserer klienten, så det giver desværre ikke så meget mening at mocke klienten her.

Dvs. der er ikke en `setHandler` e.l., så man er nødt til at bygge en sådan og skabe en ny klient i hver test. Hvilket her desværre også betyder at lave en ny `GIRUtils`-instans i hver test.

Reply:
Ah, okay, så lader vi det være.